test/invariant/handlers/TimeBasedRateHandler.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+import { StdCheats } from "forge-std/StdCheats.sol";
+import { StdUtils }  from "forge-std/StdUtils.sol";
+
+import { DSRAuthOracle } from "lib/xchain-dsr-oracle/src/DSRAuthOracle.sol";
+import { IDSROracle }    from "lib/xchain-dsr-oracle/src/interfaces/IDSROracle.sol";
+
+contract TimeBasedRateHandler is StdCheats, StdUtils {
+
+    uint256 public dsr;
+    uint256 public chi;
+    uint256 public rho;
+
+    uint256 constant ONE_HUNDRED_PCT_APY_DSR = 1.00000002197955315123915302e27;
+
+    DSRAuthOracle public dsrOracle;
+
+    uint256 public setRateCount;
+
+    constructor(DSRAuthOracle dsrOracle_) {
+        dsrOracle = dsrOracle_;
+    }
+
+    // This acts as a receiver on an L2.
+    // Note that the chi value is not derived from previous values, this is to test if
+    // PSM will work as expected with different chi values.
+    function setPotData(uint256 newDsr, uint256 newChi, uint256 newRho) external {
+        dsr = _bound(newDsr, 1e27, ONE_HUNDRED_PCT_APY_DSR);
+        chi = _bound(newChi, chi,  1e27);
```

Comment:
`chi` is a number that is always greater than `1e27`. This should be modified to reflect that.